sql/core/src/main/scala/org/apache/spark/sql/parquet/CatalystSchemaConverter.scala

```diff
@@ -490,7 +490,7 @@ private[parquet] class CatalystSchemaConverter(
           .buildGroup(repetition).as(LIST)
           .addField(
             Types.repeatedGroup()
-              .addField(convertField(StructField("element", elementType, containsNull)))
+              .addField(convertField(StructField("array", elementType, containsNull)))
```

Comment:
This line shouldn't be changed. As commented above, this case branch is implements standard Parquet schema conversion following the Parquet format spec, which explicitly require the inner most element type name to be `element`.